docs/nodes/maintain/avalanchego-config-flags.md

```diff
@@ -1308,6 +1308,10 @@ GiB).
 
 If true, runs the node as a [plugin.](https://github.com/hashicorp/go-plugin) Defaults to `false`.
 
+#### `--plugin-dir` (string)
```

Comment:
I believe there are some places refer `--build-dir` flag and `#build-directory` section in the repo. Can you update them with this flag?

Reply:
fixed a875a304a2d11e4596b23bfd31856db8772e959a

Reply:
It looks good. Just a small nit left. Thanks for noticing this missing flag!